src/sentry/search/utils.py

```diff
@@ -51,74 +51,126 @@ def parse_datetime_range(value):
         raise InvalidQuery(u'{} is not a valid datetime query'.format(value))
 
     if flag == '-':
-        return (timezone.now() - delta, None)
+        return (
+            (timezone.now() - delta, True),
+            None,
+        )
     else:
-        return (None, timezone.now() - delta)
+        return (
+            None,
+            (timezone.now() - delta, True),
+        )
```

Comment:
Making these both inclusive was sort of an arbitrary choice, I can change it if anyone has strong opinions otherwise.